### DISCUSSION TITLE
6주차 과제 - Class / Object + Data Viz

### DISCUSSION THREAD

**Discussion by youngsangcho** [youngsangcho/generativedesign2_2021#20] [Show and tell]
#과제 1
5주차 배열 과제를 Class 사용하는 코드로 변환
(추가 점수) 아이디어 및 기능 추가

1개 이상의 스크린샷 이미지

스크린 동영상 캡쳐하여 유튜브 업로드 유튜브 embed

간단한 설명 (왜 / 어떻게)

p5js editor 의 File - Share - Present 링크

p5js editor 의 File - Share - Edit 링크

코드 복사 붙여넣기

#과제2
향후 데이터 시각화 과제에 사용할 아이디어 스케치 그려보기.
일러스트레이터 / 포토샵

비쥬얼은 러프해도 OK
다른 데이터와의 조합, 인터랙션, 정렬 등 다양한 아이디어 더해보기.

**Comment by eunjinhong** [comment]
홍은진
https://blog.naver.com/tnekf1012/222548198112

**Comment by KimHyunJuhg** [comment]
김현정
https://blog.naver.com/khj317888/222547850542
